ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -60,6 +67,25 @@ export class BucketComponent implements OnInit {
     this.fetchBucketTasks();
     this.fetchBucketTasksStates();
     this.fetchBucketTaskPriorities();
+    this.fetchAssigness();
+  }
+
+  showAssigneeList() {
+    this.showAssigneeListItems = !this.showAssigneeListItems;
+  }
+
+  selectAssignee(assignee: Assignee) {
+    setTimeout(() => {
```

Comment:
setTimeout wyglada na jakis hak zeby komponent sie odswiezyl. Sprobujmy zrobic to bez setTimeout

Reply:
Jasne, działa spokojnie bez setTimeout. To efekt uboczny kodu wygenerowanego przez chatGPT, niestety często popełnia błędy :)